src/data_collection.py

```diff
@@ -269,6 +269,98 @@ def print_individual_in_table(file_name):
                 dictionary[key]["FORMAT"],
             ]
         )
+
+    print(data_table)
+
+
+def get_testing_commit_info(json_file_name):
+    """Get repo info about commits to or not to testing."""
+
+    current_data = json_handler.get_dict_from_json_file(json_file_name)
+    # creates a hashmap where the key is the authors username
+    data_dict = {}
+    # Check if RAW_DATA is in json
+    if "RAW_DATA" in current_data.keys():
+        for key in current_data["RAW_DATA"]:
+            author = key["author_name"]
+            filepaths = key["filepath"]  # get filepaths of file modified
+
+            # NOTE check date compatibility with json
+            # date = "N/A"
+            # check if the key already in in the dicitionary
+            if author in data_dict:
+                # condition passed, adds one to the number of commits
+                data_dict[author]["COMMITS"] += 1
+            else:
+                # condition fails, creates a new key and adds empty data
+                data_dict[author] = {
```

Comment:
Author keys should have already been initialized. Initializing them again will overwrite previous data on them and might cause key errors when printing them in the future.